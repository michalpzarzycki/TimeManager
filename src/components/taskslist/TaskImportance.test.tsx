import React from 'react'
import {shallow} from 'enzyme'
import TaskImportance from './TaskImportace'

test('Expext to render TaskImportance component', () => {
    expect(shallow(<TaskImportance task={{}}/>)).toMatchSnapshot()
})
