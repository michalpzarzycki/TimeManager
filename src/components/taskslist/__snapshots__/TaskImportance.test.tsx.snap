// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render TaskImportance component 1`] = `ShallowWrapper {}`;
